src/components/page-types/associatesDirectoryPage/associatesDirectoryPage.js

```diff
@@ -36,7 +36,7 @@ const AssociatesDirectoryPage = (props) => {
             </Container>
           </header>
           {hasRichText(intro) && (
-            <div className={dcnb('su-big-paragraph su-max-w-prose')}>
+            <div className={dcnb('su-big-paragraph su-max-w-prose su-mx-auto')}>
```

Comment:
```suggestion
            <div className={dcnb('su-big-paragraph su-max-w-prose su-mx-auto su-text-center')}>
```
@moisesnarvaez would you be able to make this mini update? 😄 
cc: @William-Misiaszek